### PR TITLE
Fix typo in coriolis_forces.md

### PR DESCRIPTION
**Pull request by francispoulin**
As discusse in #1591 .   Now I have to create a reference.

### CODE REVIEW

**Comment by ali-ramadhan** [comment]
Ah I think you just have to add the BibTeX entry into https://github.com/CliMA/Oceananigans.jl/blob/master/docs/oceananigans.bib and it should just work.

File: docs/oceananigans.bib [hunk 1]
```diff
@@ -294,6 +294,16 @@ @article{Buzbee70
 	pages = {627--656}
 }
 
+@article{[DellarJFM2011,
```
Comment (ali-ramadhan) [inline docs/oceananigans.bib:297]: Might work after this

```suggestion
@article{DellarJFM2011,
```

**Review by ali-ramadhan — APPROVED** [review]

File: docs/src/physics/coriolis_forces.md [hunk 2]
```diff
@@ -61,4 +61,5 @@ the locally vertical and the locally horizontal components of the rotation vecto
     \boldsymbol{f} = \left[ 2\Omega\cos\varphi_0 \left( 1 -  \frac{z}{R} \right) + \gamma y \right] \boldsymbol{\hat y}
            + \left[ 2\Omega\sin\varphi_0 \left( 1 + 2\frac{z}{R} \right) + \beta  y \right] \boldsymbol{\hat z} \, ,
 ```
-where ``\beta = 2 \Omega \cos \varphi_0 / R`` and ``\gamma = -4 \Omega \varphi_0 / R``.
+as can be found in [DellarJFM2011](@cite) where
```
Comment (navidcy) [inline docs/src/physics/coriolis_forces.md:64]: "in Dellar et al. 2011" reads as you are referring to Bill's internal organs :)

perhaps better is "in the work by Dellar..."?

[I know that this is a common way to phrase things in scientific literature, i.e., omitting "the work by" or "the paper by" and just saying "in Navid et al" but still it does not read well to me...]

**Review by navidcy — APPROVED** [review]

File: docs/src/physics/coriolis_forces.md (hunk 2 above)
Comment (navidcy) [reply docs/src/physics/coriolis_forces.md:64]: Perhaps also a comma after the citation?
" Dellar 2011, where..."?

**Comment by navidcy** [comment]
https://clima.github.io/OceananigansDocumentation/previews/PR1592/physics/coriolis_forces/

File: docs/src/physics/coriolis_forces.md (hunk 2 above)
Comment (francispoulin) [reply docs/src/physics/coriolis_forces.md]: Hahaha, I agree with you completely.  It is standard and could imply we are discussing his organs.  I like "in the paper by".  Thanks for the suggestion and will change it right now and push it.

**Pull request merged**